canaryconfigmgr/canaryConfigMgr.go

```diff
@@ -115,17 +122,21 @@ func (canaryCfgMgr *canaryConfigMgr) initCanaryConfigController() (k8sCache.Stor
 
 func (canaryCfgMgr *canaryConfigMgr) Run(ctx context.Context) {
 	go canaryCfgMgr.canaryConfigController.Run(ctx.Done())
-	log.Printf("started Canary configmgr controller")
+	canaryCfgMgr.logger.Info("started canary configmgr controller")
 }
 
 func (canaryCfgMgr *canaryConfigMgr) addCanaryConfig(canaryConfig *crd.CanaryConfig) {
-	log.Printf("addCanaryConfig called for %s", canaryConfig.Metadata.Name)
+	canaryCfgMgr.logger.Info("addCanaryConfig called", zap.String("called_for", canaryConfig.Metadata.Name))
```

Comment:
shouldn't it be something like "canary_config" or even just "name" instead of "called_for"?

Reply:
yup, I'll fix that